docs/mmteb/create_points_table.py

```diff
@@ -6,7 +6,7 @@
 import pandas as pd
 
 
-def load_data(file_path: Path) -> pd.DataFrame:
```

Comment:
This causes the test to fail: https://github.com/embeddings-benchmark/mteb/actions/runs/10652807489/job/29527177315